src/payment/checkout/payment-form/StripePaymentForm.jsx

```diff
@@ -128,7 +129,13 @@ function StripePaymentForm({
       } else {
         // Otherwise send paymentIntent.id to your server
         // TODO: refactor to fetch in service.js
-        const postData = formurlencoded({ payment_intent_id: result.paymentIntent.id });
+
+        // generate comma separated list of product SKUs
+        const skus = products.map(({ sku }) => sku).join(',');
```

Comment:
What data structure is `skus` in this case? If I am reading correctly, it will be a string with all the `sku` (if bundle) separated by a comma? That seems funny to me, but I am assuming that's what the BE expects

Reply:
I clearly missed your comment 😅, I see you mention a "list" but I think this ends up being a string. Did you meant to create an array instead?

Reply:
yeah skus would be a string that has comma separated sku ids (e.g. `AB138E,DEF918`). historically i've found strings easier to deal with when passing to an endpoint, but also i originally had a list that was being urlencoded and the key in the params that made it to ecommerce was `'skus[0]'`. the `[0]` seemed weird to me, and comma separated strings are usually okay as well, so I went with that

but yeah, i meant lower case "l" list, the concept, not the data structure ;)